docs/developer/quickstart.md

```diff
@@ -153,3 +175,9 @@ make generate-njs-yaml
 Additionally, the [NJS ConfigMap Helm template](/deploy/helm-chart/templates/njs-modules.yaml) will need to be updated.
 This is currently a manual process - ensure the content in the `data` field matches that in the
 [NJS ConfigMap manifest](/deploy/manifests/njs-modules.yaml) `data` field.
+
+Finally, to update the combined generated manifest, run the following make command from the project's root directory:
```

Comment:
since we generate multiple files
```suggestion
Finally, to update the all other generated manifests, run the following make command from the project's root directory:
```